packages/@stimulus/core/src/class_map.ts

```diff
@@ -12,7 +12,17 @@ export class ClassMap {
   }
 
   get(name: string) {
-    return this.data.get(this.getDataKey(name))
+    return this.getAll(name)[0];
+  }
+
+  getAll(name: string) {
+    const classAttribute = this.data.get(this.getDataKey(name));
+
+    if (classAttribute) {
+      return classAttribute.split(" ")
```

Comment:
`tokenString` might be a better name for this variable, and I'd trim + split on white space. Following the implementation here https://github.com/stimulusjs/stimulus/blob/8cb58d1f8a4ac83a875c7a1fdc7a28f21869efd6/packages/%40stimulus/mutation-observers/src/token_list_observer.ts#L106 this method could be rewritten as
```js
getAll(name: string) {
  const tokenString = this.data.get(this.getDataKey(name)) || ""
  return tokenString.trim().split(/\s+/)
}
```

Reply:
> this method could be rewritten as
> ```js
> getAll(name: string) {
>   const tokenString = this.data.get(this.getDataKey(name)) || ""
>   return tokenString.trim().split(/\s+/)
> }
> ```

Good feedback -- just to clarify, we'll still need the `filter` otherwise we will return `[""]` if the controller is missing the class attribute.